frontend/src/app/modules/user/profile-page-main-info/profile-page-main-info.component.sass

```diff
@@ -186,3 +186,52 @@
     display: inline-block
     width: fit-content
     color: $white
+
+.dot-flashing
```

Comment:
It may be better to separate this styling and use it both here and in leader board to avoid duplication